leptonai/cli/photon.py

```diff
@@ -66,7 +66,9 @@ def _get_most_recent_photon_id_or_none(conn: Connection, name: str) -> Optional[
     return photon_ids[0] if photon_ids else None
 
 
-def _add_workspace_token_secret_var_if_not_existing(conn: Connection):
+def _add_workspace_token_secret_var_if_not_existing(
```

Comment:
for clarity we can call it "create_workspace_token_secret_var_if_not_existing"

Reply:
done